### ISSUE TITLE
Intermittent timeout in /html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html

### ISSUE THREAD

**Issue by jdm** [servo/servo#22007]
```
  ▶ TIMEOUT [expected OK] /html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html
  │ 
  │ VMware, Inc.
  │ softpipe
  │ 3.3 (Core Profile) Mesa 18.1.0-devel
  │ 0:00:00.029868393 26188 0x7f1bb402d230 WARN                typefind gsttypefindelement.c:856:gst_type_find_get_extension:<typefind> could not find uri extension in appsrc://
  │ 0:00:00.029878465 26188 0x7f1ba40039e0 WARN                typefind gsttypefindelement.c:856:gst_type_find_get_extension:<typefind> could not find uri extension in appsrc://
  │ 0:00:00.041783470 26188 0x7f1ba40039e0 FIXME           videodecoder gstvideodecoder.c:944:gst_video_decoder_drain_out:<theoradec0> Sub-class should implement drain()
  │ 0:00:00.043121526 26188 0x7f1ba40039e0 WARN                 default oss4-property-probe.c:303:gst_oss4_property_probe_get_values:<oss4sink0> Can't open file descriptor to probe available devices: No such file or directory
  │ 0:00:00.043155941 26188 0x7f1ba40039e0 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<oss4sink0> error: Could not open audio device for playback.
  │ 0:00:00.043166662 26188 0x7f1ba40039e0 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<oss4sink0> error: system error: No such file or directory
  │ 0:00:00.043344953 26188 0x7f1ba40039e0 WARN                 playbin gstplaybin2.c:4663:autoplug_select_cb:<playbin> Could not activate sink oss4sink
  │ 0:00:00.043905892 26188 0x7f1ba40039e0 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<osssink0> error: Could not open audio device for playback.
  │ 0:00:00.043917270 26188 0x7f1ba40039e0 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<osssink0> error: system error: No such file or directory
  │ 0:00:00.043956238 26188 0x7f1ba40039e0 WARN                 playbin gstplaybin2.c:4663:autoplug_select_cb:<playbin> Could not activate sink osssink
  │ 0:00:00.044619555 26188 0x7f1bb402d230 WARN                 default oss4-property-probe.c:303:gst_oss4_property_probe_get_values:<oss4sink1> Can't open file descriptor to probe available devices: No such file or directory
  │ 0:00:00.044652676 26188 0x7f1bb402d230 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<oss4sink1> error: Could not open audio device for playback.
  │ 0:00:00.044660546 26188 0x7f1bb402d230 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<oss4sink1> error: system error: No such file or directory
  │ 0:00:00.044710005 26188 0x7f1bb402d230 WARN                 playbin gstplaybin2.c:4663:autoplug_select_cb:<playbin> Could not activate sink oss4sink
  │ 0:00:00.044789311 26188 0x7f1bb402d230 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<osssink1> error: Could not open audio device for playback.
  │ 0:00:00.044796571 26188 0x7f1bb402d230 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<osssink1> error: system error: No such file or directory
  │ 0:00:00.044837503 26188 0x7f1bb402d230 WARN                 playbin gstplaybin2.c:4663:autoplug_select_cb:<playbin> Could not activate sink osssink
  │ 0:00:00.045636482 26188 0x7f1b9c0328f0 FIXME           videodecoder gstvideodecoder.c:944:gst_video_decoder_drain_out:<theoradec0> Sub-class should implement drain()
  │ 0:00:00.048932446 26188 0x7f1b9c032e30 WARN                 default oss4-property-probe.c:303:gst_oss4_property_probe_get_values:<audiosink-actual-sink-oss4> Can't open file descriptor to probe available devices: No such file or directory
  │ 0:00:00.048964471 26188 0x7f1b9c032e30 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<audiosink-actual-sink-oss4> error: Could not open audio device for playback.
  │ 0:00:00.048971450 26188 0x7f1b9c032e30 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<audiosink-actual-sink-oss4> error: system error: No such file or directory
  │ 0:00:00.049082723 26188 0x7f1b9c032e30 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<audiosink-actual-sink-oss> error: Could not open audio device for playback.
  │ 0:00:00.049089439 26188 0x7f1b9c032e30 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<audiosink-actual-sink-oss> error: system error: No such file or directory
  │ 0:00:00.055147765 26188 0x7f1c1c4085e0 FIXME                    bin gstbin.c:4337:gst_bin_query: implement duration caching in GstBin again
  │ 0:00:00.066441179 26188 0x7f1b9c0328f0 WARN                 default oss4-property-probe.c:303:gst_oss4_property_probe_get_values:<audiosink-actual-sink-oss4> Can't open file descriptor to probe available devices: No such file or directory
  │ 0:00:00.066473393 26188 0x7f1b9c0328f0 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<audiosink-actual-sink-oss4> error: Could not open audio device for playback.
  │ 0:00:00.066478680 26188 0x7f1b9c0328f0 WARN                oss4sink oss4-sink.c:514:gst_oss4_sink_open:<audiosink-actual-sink-oss4> error: system error: No such file or directory
  │ 0:00:00.066554117 26188 0x7f1b9c0328f0 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<audiosink-actual-sink-oss> error: Could not open audio device for playback.
  └ 0:00:00.066559842 26188 0x7f1b9c0328f0 WARN                     oss gstosssink.c:399:gst_oss_sink_open:<audiosink-actual-sink-oss> error: system error: No such file or directory

  ▶ Unexpected subtest result in /html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html:
  └ NOTRUN [expected FAIL] calling play() on a sufficiently long audio should trigger timeupdate event

  ▶ Unexpected subtest result in /html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html:
  └ NOTRUN [expected FAIL] calling play() on a sufficiently long video should trigger timeupdate event
```

**Comment by jdm** [comment]
This appeared in #21999.

**Comment by ferjm** [comment]
There are also intermittents for the non autoplay version of this test:

```
{
    "status": "PASS", 
    "group": "default", 
    "message": null, 
    "stack": null, 
    "subtest": "setting src attribute on a sufficiently long autoplay audio should trigger timeupdate event", 
    "test": "/html/semantics/embedded-content/media-elements/event_timeupdate.html", 
    "line": 202692, 
    "action": "test_result", 
    "expected": "NOTRUN"
}
```

This will hopefully be fixed by #22477

**Comment by ferjm** [comment]
#22477 has just landed and I believe it should fix this intermittent. Please, reopen if you see the timeouts again.
